app/pages/sections/routes.js

```diff
@@ -41,7 +42,10 @@ export const sectionRoutes = (authProvider, addContext, sessionManager) => {
       accessToken,
     });
 
-    if (response.success) return res.redirect(`${config.baseUrl}/organisation/${response.orderId}`);
+    if (response.success) {
+      sessionManager.saveToSession({ req, key: 'description', value: req.body.description.trim() });
```

Comment:
I've created a `sessionKeys` object as part of task 8868 (PR currently awaiting review). Assuming we merge that PR before this one then I'd recommend we use that for the session key here.

It looks like we've used `orderDescription` as the session key elsewhere but that's irrelevant if we do what I've suggested above – unless there's a specific reason for the use of a different key here?